### ISSUE TITLE
JIT disabled caused by fiber.

### ISSUE THREAD

**Issue by juzi5201314** [amphp/ext-fiber#23]
No warning appeared when I disabled ext-fiber, but when I enabled fiber:  `warning: JIT is incompatible with third party extensions that setup user opcode handlers. JIT disabled`.

I used the elimination method, and finally determined that disabling fiber can eliminate this warning.

my modules:
```
[PHP Modules]
Core
ctype
curl
date
dom
fiber
fileinfo
filter
hash
iconv
json
libxml
mbstring
openssl
parallel
pcre
PDO
pdo_pgsql
pdo_sqlite
Phar
posix
Reflection
session
SimpleXML
sockets
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
Zend OPcache
zlib

[Zend Modules]
Zend OPcache
```

php version:
```
PHP 8.1.0-dev (cli) (built: Apr 16 2021 18:02:43) ( ZTS )
Copyright (c) The PHP Group
Zend Engine v4.1.0-dev, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.0-dev, Copyright (c), by Zend Technologies
```

os: `Ubuntu20 on wsl`

**Comment by kelunik** [comment]
This won't be an issue once it's merged into PHP 8.1.

**Comment by juzi5201314** [comment]
> 一旦将其合并到PHP 8.1中，这将不再是问题。

ok
